backend/src/nodes/nodes/image/create_noise.py

```diff
@@ -0,0 +1,198 @@
+from nodes.impl.noise_functions.value import ValueNoise
+from nodes.node_base import NodeBase, group
+from nodes.node_factory import NodeFactory
+from nodes.properties import expression
+from nodes.properties.inputs import (
+    NumberInput,
+    EnumInput,
+    SliderInput,
+    BoolInput,
+)
+from nodes.properties.outputs import ImageOutput
+
+
+class NoiseMethod(Enum):
+    VALUE = "Value Noise"
+    SIMPLEX = "Simplex"
+
+
+class FractalMethod(Enum):
+    NONE = "None"
+    PINK = "Pink noise"
+
+
+@NodeFactory.register("chainner:image:create_noise")
+class CreateNoiseNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Create an image of specified dimensions filled with one of a variety of noises."
+        self.inputs = [
+            NumberInput("Width", minimum=1, unit="px", default=256),
+            NumberInput("Height", minimum=1, unit="px", default=256),
+            group("seed")(
+                NumberInput("Seed", minimum=0, maximum=2**32 - 1, default=0),
+            ),
+            EnumInput(
+                NoiseMethod,
+                default_value=NoiseMethod.SIMPLEX,
+                option_labels={key: key.value for key in NoiseMethod},
+            ).with_id(3),
+            NumberInput("Scale", minimum=1, default=50, precision=1).with_id(4),
+            SliderInput(
+                "Brightness", minimum=0, default=100, maximum=100, precision=2
+            ).with_id(5),
+            BoolInput("Tile Horizontal", default=False).with_id(10),
+            BoolInput("Tile Vertical", default=False).with_id(11),
+            BoolInput("Tile Spherical", default=False).with_id(12),
+            EnumInput(
+                FractalMethod,
+                default_value=FractalMethod.NONE,
+                option_labels={key: key.value for key in FractalMethod},
+            ).with_id(6),
+            group(
+                "conditional-enum",
+                {
+                    "enum": 6,
+                    "conditions": [
+                        FractalMethod.PINK.value,
+                        FractalMethod.PINK.value,
+                        FractalMethod.PINK.value,
+                        FractalMethod.PINK.value,
+                    ],
+                },
+            )(
+                NumberInput("Layers", minimum=2, default=3, precision=1).with_id(7),
+                NumberInput("Scale Ratio", minimum=1, default=2, precision=2).with_id(
+                    8
+                ),
+                NumberInput(
+                    "Brightness Ratio", minimum=1, default=2, precision=2
+                ).with_id(9),
+                BoolInput("Increment Seed", default=True).with_id(13),
+            ),
+        ]
+        self.outputs = [
+            ImageOutput(
+                image_type=expression.Image(
+                    width="Input0",
+                    height="Input1",
+                    channels="1",
+                )
```

Comment:
Using the channels parameter on `ImageOutput` is preferred.
```suggestion
                image_type=expression.Image(
                    width="Input0",
                    height="Input1",
                ),
                channels=1,
```